modular_skyrat/modules/modular_weapons/code/company_and_or_faction_based/carwo_defense_systems/magazines.dm

```diff
@@ -0,0 +1,134 @@
+//
+// .35 Sol pistol magazines
+//
+
+/obj/item/ammo_box/magazine/c35sol_pistol
+	name = "\improper Sol pistol magazine"
+	desc = "A standard size magazine for SolFed pistols, holds twelve rounds."
+
+	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/company_and_or_faction_based/carwo_defense_systems/ammo.dmi'
+	icon_state = "pistol_35_standard"
+
+	multiple_sprites = AMMO_BOX_FULL_EMPTY
+
+	w_class = WEIGHT_CLASS_TINY
+
+	ammo_type = /obj/item/ammo_casing/c35sol
+	caliber = CALIBER_SOL35SHORT
+	max_ammo = 12
+
+/obj/item/ammo_box/magazine/c35sol_pistol/starts_empty
+	start_empty = TRUE
+
+/obj/item/ammo_box/magazine/c35sol_pistol/stendo
+	name = "\improper Sol extended pistol magazine"
+	desc = "An extended magazine for SolFed pistols, holds twenty-four rounds."
+
+	icon_state = "pistol_35_stended"
+
+	w_class = WEIGHT_CLASS_NORMAL
+
+	max_ammo = 24
+
+/obj/item/ammo_box/magazine/c35sol_pistol/stendo/starts_empty
+	start_empty = TRUE
+
+//
+// .40 Sol rifle magazines
+//
```

Comment:
Your style of comment isn't consistent between the files in this PR. Pick one and stick to it.